### DISCUSSION TITLE
I want to run form with select and then return download file

### DISCUSSION THREAD

**Discussion by uluumbch** [orchidsoftware/platform#2738] [Q&A]
hi everyone, is it possible to return response()->download($filename)->deleteFileAfterSend(true); in screen method, I tried many ways but it doesnt work. Even when I want to create my own form using custom Layout::view, this seem the turbo overrride my form. please any help will be appreciate

**Comment by tabuna** [comment]
Add a method to the button:

Button:make('Download')->rawClick()

Or (for later versions) 

Button::make('Download')->download()

**Reply by uluumbch** [reply]
I want user to select month first before download the file, I tried using modal and field but it doesn't seem work
